### PR TITLE
Create a voting procedure

### PR DESCRIPTION
**Pull request by mattcaswell**
This is based on the OTC voting procedure with minor updates OTC->OMC

### CODE REVIEW

File: policies/voting-procedure.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  OMC Member B  [  ]
+  ...
+```
+
+The individual member votes are recorded as `[+1]` a vote in favour, `[-1]`
+a vote against, `[+0]` an abstention with an inclination in favour,
+`[ 0]` a neutral abstention, `[-0]` an abstention with an inclination
+against, and `[  ]` meaning not voted.
+
+The vote files are named `yyyymmdd-vote-short-id.txt` where the `yyyymmdd`
```
Comment (levitte) [inline policies/voting-procedure.md:59]: So, in technical policies, I see files called `vote-yyyymmdd-vote-short-id.txt`...  are we going to allow those too?  Might as well be consistent in our inconsistency...

Comment (mattcaswell) [reply policies/voting-procedure.md]: I've change the filename format in this policy to say `vote-yyyymmdd-vote-short-id.txt` as per current OTC practice. I've also proposed an equivalent change to the OTC policy:
https://github.com/openssl/technical-policies/pull/17

**Review by paulidale — APPROVED** [review]

**Comment by mattcaswell** [comment]
Vote for accepting the policy at the commit id 7d641f5319e3 has started.

**Comment by mattcaswell** [comment]
Vote: [+1]

**Comment by iamamoose** [comment]
[+1]

**Comment by kroeckx** [comment]
+1

**Comment by paulidale** [comment]
Vote: [+1]

**Comment by t-j-h** [comment]
Vote: [+1]

**Comment by levitte** [comment]
vote: [+1]

**Comment by mattcaswell** [comment]
Closing the vote now:

````
Topic: Accept openssl/general-policies PR#1 - the policy change
       process proposal as of commit 7d641f5. This will become an official OMC
       policy.
Proposed by: Matt Caswell
Issue link: https://github.com/openssl/general-policies/pull/1
Public: yes
Opened: 2022-01-25
Closed: 2022-02-02
Accepted:  yes  (for: 6, against: 0, abstained: 0, not voted: 0)

  Matt       [+1]
  Mark       [+1]
  Pauli      [+1]
  Tim        [+1]
  Richard    [+1]
  Kurt       [+1]
````

**Pull request closed without merging**